bindings/node.js/src/kzg.cxx

```diff
@@ -490,11 +513,15 @@ Napi::Value VerifyBlobKzgProofBatch(const Napi::CallbackInfo &info) {
         // add HandleScope here to release reference to temp values
         // after each iteration since data is being memcpy
         Napi::HandleScope scope{env};
-        Blob *blob = get_blob(env, blobs_param[index]);
+        uint8_t *blob = get_blob(env, kzg_settings, blobs_param[index]);
         if (blob == nullptr) {
             goto out;
         }
-        memcpy(&blobs[index], blob, BYTES_PER_BLOB);
+        memcpy(
+            &blobs[index * kzg_settings->bytes_per_blob],
```

Comment:
Hmm, I see we don't define a new custom `Blob` type as we do in the other bindings and here the memcpy index logic has to change. Is this intentional?

Reply:
To my knowledge, like in C, it's not very practical in C++ to have a custom `Blob` type. And yes, the index logic needs to change since it's changed from an array of blobs (known size) to an array of bytes. The two should be equivalent. Given some bytes, to get to the start of the blob N, you must jump `N * bytes_per_blob` into the array.

Reply:
So I added back the `Blob` type in `lib/kzg.d.ts`. This is akin to how other bindings do it.

Reply:
The Blob struct will no longer exist. It's been removed in that PR.